extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -8,8 +8,8 @@ import * as vscode from 'vscode';
 import { getErrorMessage } from '../utils';
 import * as localizedConstants from './localizedConstants';
 
-export async function refreshParentNode(context: azdata.ObjectExplorerContext): Promise<void> {
-	if (context) {
+export async function refreshParentNode(context: azdata.ObjectExplorerContext, isRootNode?: boolean): Promise<void> {
```

Comment:
This should probably not be optional - all nodes should be either root or not, right? We definitely don't want to risk missing a call and having it not have the correct node type. 

And TBH - given that parentUrn is calculated from the OE context itself already it seems like you should be able to determine whether this is a root node here without needing to pass in another variable.